dashboard/app/models/game.rb

```diff
@@ -127,7 +127,7 @@ def uses_pusher?
   end
 
   def uses_small_footer?
-    app == NETSIM || app == APPLAB
+    app == NETSIM || app == APPLAB || name == "TextCompression"
```

Comment:
If you add a `TEXT_COMPRESSION = 'text_compression'` constant you can follow the existing pattern here.